### PR TITLE
Update asciidoctor image to latest released versions

### PR DESCRIPTION
**Pull request by buuhsmead**
#### What is this PR About?
Describe the contents of the PR
- Updates the images to the latest released asciidoctor products
- removed kindlegen product because upstream (asciidoctor did this also) 
- update version.json from 1.1 into 1.2

#### How do we test this?
Provide commands/steps to test this PR.
Create the image and check the output of the pdf ....
docker run --rm --name asciidoctor -v /<report-dir>/:/documents/ -w /documents/styles quay.io/redhat-cop/ubi8-asciidoctor:1.2 asciidoctor-pdf -r asciidoctor-diagram -a ej-base-dir=/documents/  <document.adoc>
cc: @redhat-cop/day-in-the-life

### CODE REVIEW

File: utilities/ubi8-asciidoctor/Dockerfile [hunk 1]
```diff
@@ -39,11 +39,12 @@ RUN microdnf install \
...
   gcc-c++ \
-  glib2-devel
+  glib2-devel \
+  && microdnf clean all
 
 # install pandoc
-RUN curl -L https://github.com/jgm/pandoc/releases/download/${PANDOC_VERSION}/pandoc-${PANDOC_VERSION}-linux-amd64.tar.gz -o pandoc-${PANDOC_VERSION}-linux-amd64.tar.gz
-RUN tar xvzf pandoc-${PANDOC_VERSION}-linux-amd64.tar.gz --strip-components 1 -C /usr/local/
+RUN curl -L https://github.com/jgm/pandoc/releases/download/${PANDOC_VERSION}/pandoc-${PANDOC_VERSION}-linux-amd64.tar.gz -o pandoc-${PANDOC_VERSION}-linux-amd64.tar.gz 
+RUN  tar xvzf pandoc-${PANDOC_VERSION}-linux-amd64.tar.gz --strip-components 1 -C /usr/local/
```
Comment (pabrahamsson) [inline utilities/ubi8-asciidoctor/Dockerfile:47]: Please remove added whitespace

**Review by pabrahamsson — CHANGES_REQUESTED** [review]
Looks good @buuhsmead, minor cleanup request inline.

File: utilities/ubi8-asciidoctor/Dockerfile (hunk 1 above)
Comment (buuhsmead) [reply utilities/ubi8-asciidoctor/Dockerfile]: Adjusted as requested.

File: utilities/ubi8-asciidoctor/Dockerfile [hunk 2]
```diff
@@ -39,11 +39,12 @@ RUN microdnf install \
...
   git \
   gcc-c++ \
-  glib2-devel
+  glib2-devel \
+  && microdnf clean all
 
 # install pandoc
 RUN curl -L https://github.com/jgm/pandoc/releases/download/${PANDOC_VERSION}/pandoc-${PANDOC_VERSION}-linux-amd64.tar.gz -o pandoc-${PANDOC_VERSION}-linux-amd64.tar.gz
-RUN tar xvzf pandoc-${PANDOC_VERSION}-linux-amd64.tar.gz --strip-components 1 -C /usr/local/
+RUN tar xvzf pandoc-${PANDOC_VERSION}-linux-amd64.tar.gz --strip-components 1 -C /usr/local/
```
Comment (pabrahamsson) [inline utilities/ubi8-asciidoctor/Dockerfile:47]: Sorry @buuhsmead, I missed this trailing space the first time

**Review by pabrahamsson — CHANGES_REQUESTED** [review]

File: utilities/ubi8-asciidoctor/Dockerfile (hunk 2 above)
Comment (buuhsmead) [reply utilities/ubi8-asciidoctor/Dockerfile]: No Problem, have to adjust several editors to remove at least spaces at the end of a line.

**Review by pabrahamsson — APPROVED** [review]
Thank you @buuhsmead

**Pull request merged**